custom_components/enpal/config_flow.py

```diff
@@ -16,9 +16,10 @@
 big_int = vol.All(vol.Coerce(int), vol.Range(min=300))
 
 _LOGGER = logging.getLogger(__name__)
+
 CONFIG_SCHEMA = vol.Schema(
             {
-                vol.Required('enpal_host_ip', default='192.168.178.'): cv.string,
+                vol.Required('enpal_host_ip', default='192.168.137.36'): cv.string,
```

Comment:
Putting your own IP as default for everyone - I don't like this idea. 
If someone will forget to change it or won't know that it have to be changed, stuff will not work.

192.168.178. - that's not complete, so you know it is to be edited and it's most popular IP pattern in Germany.

Reply:
yes that was my mistake -.-